omniNotes/src/androidTest/java/it/feio/android/omninotes/tests/HamburgerTest.java

```diff
@@ -19,37 +24,56 @@ public void HamburgerViewTest() {
         AddTextNote.typePayTheBills();
 
         // Step: Click return button
+        AddTextNote.clickBackButton();
 
         // Step: Delete (swipe) the note with title "Pay the bills"
+        Home.deletePayTheBillsNote();
 
         // Step: Click on hamburger
+        AddTextNote.clickBackButton();
 
         // Verify: The button with name Archive is displayed and has sibling the button with name Notes
+        assertTrue("The button with name Archive is not displayed", Home.isArchiveButtonDisplayed());
+        assertTrue("The button with name Archive doesn't have sibling the button with name Notes", Home.isArchiveButtonSiblingWithNotesButton());
```

Comment:
This message is to long and confusing. You can write "Archive button is not sibling with Notes button". Is more simple to understand.